.travis.yml

```diff
@@ -28,7 +28,7 @@ after_failure:
 notifications:
   email:
     recipients:
-      - stream-registry-committers@groups.homeawaycorp.com
+      - stream-registry-committers@expediagroup.com
```

Comment:
Not sure you want to put this email address in the public domain. Travis is pretty good at informing the person who did the commit if the build failed and it also shows up in the pull request etc. Up to you guys.

Reply:
I agree. We can revisit this later. I was purely focussing on things moved over rather than changing anything. Not too much intelligence applied. Mostly a massive find+replace.